src/components/button/button.styled.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+const StyledButton = styled.div<{
+  background?: string;
+  color?: string;
+  radius?: number;
+  border?: string;
+  padding?: string;
+  margin?: string;
+  weight?: string;
+  size?: number;
+}>`
+  margin: ${(props) => (props.radius ? `${props.radius}` : '')};
```

Comment:
diganti aja kalo empty

```suggestion
  margin: ${(props) => (props.radius && props.radius)};
```